File: src/main/java/it/contrader/dto/TeamDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class TeamDTO {
	private long id;
	private String name;
	private int market_value;
	private int team_index;
}

File: src/dto/teamdto.ts
export class TeamDTO {
	id:number;

	name:string;

	market_value:number;

	team_index:number;

	constructor(id: number, name: string, market_value: number,  team_index: number) {
        this.id = id;
        this.name = name;
        this.market_value = market_value;
        this.team_index = team_index;
}
}
